From 689c4da0323e61840ae713f31f0134abfcf96bec Mon Sep 17 00:00:00 2001
From: Jiri Olsa <jolsa@redhat.com>
Date: Mon, 8 Sep 2014 13:07:16 -0400
Subject: [tools] perf: Remove asmlinkage define from perf.h

Message-id: <1410181752-25631-192-git-send-email-jolsa@redhat.com>
Patchwork-id: 91814
O-Subject: [PATCH RHEL7.1 BZ1134356 191/307] perf tools: Remove asmlinkage define from perf.h
Bugzilla: 1134356
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Don Zickus <dzickus@redhat.com>

From: Jiri Olsa <jolsa@kernel.org>

Bugzilla: 1134356
https://bugzilla.redhat.com/show_bug.cgi?id=1134356

upstream
========
commit 273a0a734cd780f9ab3c3aac1851b2d15bb9638d
Author: Jiri Olsa <jolsa@kernel.org>
Date: Mon May 5 12:36:38 2014 +0200

description
===========
As it's no longer used.
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/tools/perf/perf.h b/tools/perf/perf.h
index 50aa88d..622949d 100644
--- a/tools/perf/perf.h
+++ b/tools/perf/perf.h
@@ -198,11 +198,6 @@ static inline unsigned long long rdclock(void)
  return ts.tv_sec * 1000000000ULL + ts.tv_nsec;
 }
 
-/*
- * Pick up some kernel type conventions:
- */
-#define asmlinkage
-
 extern bool test_attr__enabled;
 void test_attr__init(void);
 void test_attr__open(struct perf_event_attr *attr, pid_t pid, int cpu,
-- 
1.7.1